src/index.html

```diff
@@ -30,6 +30,50 @@ <h1 class="home-hero-title">A new home for modern and equitable transportation i
   </div>
 </div>
 
+<div class="bg-dark-tan">
+  <div class="container">
+    <section class="py-5">
+      <header>
+        <h2 class="text-center pt-3">Modernize your fleet</h2>
+      </header>
+      <div class="row justify-content-center">
+        <div class="col-md-8 text-center">
+          <p>
+            Cal-ITP provides resource materials and technical assistance to public transportation providers in California
+            and beyond as they modernize their fleets with <a href="/go-contactless">contactless payments</a>, <a href="/how-to/get-connected">data plans</a>, and more. These are just some
+            of the agencies we’ve worked with.
+          </p>
+        </div>
+      </div>
+
+      <div class="go-contactless-agency-container">
+        <div class="agency-card">
+          <img alt="Clean Air Express" src="/uploads/clean-air-express.png" />
+        </div>
+        <div class="agency-card">
+          <img alt="Monterey-Salinas Transit" src="/uploads/monterey-salinas-transit-mst.png" />
+        </div>
+        <div class="agency-card">
+          <img alt="Sacramento Regional Transit" src="/uploads/sacramento-regional-transit-sacrt.png" />
+        </div>
+        <div class="agency-card">
+          <img
+            alt="Santa Barbara Metropolitan Transit District"
+            src="/uploads/santa-barbara-metropolitan-transit-district-sb-mtd.png"
+          />
+        </div>
+      </div>
```

Comment:
Since this segment of code is the same on both pages, it could be turned into an includes.

Reply:
Extracted it to an includes in https://github.com/cal-itp/mobility-marketplace/pull/491/commits/121283eb94e705a0667509e610b1693c30088be7